java/src/jmri/jmrit/roster/swing/rostergroup/RosterGroupTableAction.java

```diff
@@ -72,7 +77,11 @@ void extras() {
                 selectCombo.addActionListener(new ActionListener() {
                     @Override
                     public void actionPerformed(ActionEvent e) {
-                        comboSelected(e, selectCombo.getSelectedItem().toString());
```

Comment:
I suppose that the NPE here comes if `selectCombo` has no selected item. Another possible solution is to check if `getSelectedIndex() != -1`. For me, that would be better, since it expains why it happens.
https://docs.oracle.com/javase/7/docs/api/javax/swing/JComboBox.html#getSelectedIndex()

```
if (getSelectedIndex() != -1) {
    comboSelected(e, selectCombo.getSelectedItem().toString());
}
```